website/static/js/comment.js

```diff
@@ -16,6 +16,77 @@ var osfHelpers = require('js/osfHelpers');
 var CommentPane = require('js/commentpane');
 var markdown = require('js/markdown');
 
+var caret = require('Caret.js');
+var atWho = require('At.js');
+
+// @ mention prototyping
+var callbacks = {
+    beforeInsert: function(value, $li) {
+        var data = $li.data('item-data');
+        var model = ko.dataFor(this.$inputor[0]);
+        if (model.replyMentions().indexOf(data.id) === -1) {
+            model.replyMentions().push(data.id);
+        }
+        this.query.el.attr('data-atwho-guid', '' + data.id);
+        return value;
+    }
+};
+
+var at_config = {
+    at: '@',
+    headerTpl: '<div class="atwho-header">Contributors<small>&nbsp;↑&nbsp;↓&nbsp;</small></div>',
+    insertTpl: '@${name}',
+    displayTpl: '<li>${name} <small>${fullName}</small></li>',
+    limit: 6,
+    callbacks: callbacks
+};
+
+var plus_config = {
+    at: '+',
+    headerTpl: '<div class="atwho-header">Contributors<small>&nbsp;↑&nbsp;↓&nbsp;</small></div>',
+    insertTpl: '+${name}',
+    displayTpl: '<li>${name} <small>${fullName}</small></li>',
```

Comment:
It might be worth factoring out `headerTpl` and `displayTpl` since they're the same between the two configs. Not sure how to [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) it up further than that.